src/corelibs/U2Core/src/gobjects/MultipleChromatogramAlignmentObject.cpp

```diff
@@ -260,6 +260,47 @@ void MultipleChromatogramAlignmentObject::trimRow(const int rowIndex, int curren
     updateCachedMultipleAlignment(modificationInfo);
 }
 
+void MultipleChromatogramAlignmentObject::updateAlternativeMutations(const McaReadsTabSettings& settings, U2OpStatus& os) {
+    for (int i = 0; i < getNumRows(); i++) {
+        const MultipleChromatogramAlignmentRow& mcaRow = static_cast<const MultipleChromatogramAlignmentRow&>(getRow(i));
+        const int ungappedLength = mcaRow->getUngappedLength();
+
+        QMap<qint64, char> newCharList;
+        for (int j = 0; j < ungappedLength; j++) {
+            auto res = mcaRow->getTwoHighestPeaks(j);
+            const double minimumThresholdValue = (double)res.second.value / res.first.value * 100;
```

Comment:
nit: + same 'const' madeness here. You do not use a consistent style with const even in this method (see gappedPos or modifiedRowId below) so it is better not to use const for locals with no important reason at all. 

More than that other UGENE code does not follow the convention of 'const everywhere' so we have 2 code styles across our code which is bad: we should use 1 unified code style and 'const everywhere' breaks it.

Reply:
OK, but we don't have any agreements about it.